GitUI/Theming/ThemeRepository.cs

```diff
@@ -71,9 +86,19 @@ private IEnumerable<ThemeId> GetUserCustomizedThemeIds()
                 : Enumerable.Empty<ThemeId>();
         }
 
-        private string GetPath(ThemeId id) =>
-            Path.Combine(id.IsBuiltin
-                ? AppDirectory
-                : DataDirectory, id.Name + Extension);
+        private string GetPath(ThemeId id)
+        {
+            if (id.IsBuiltin)
+            {
+                return Path.Combine(AppThemesDirectory, id.Name + Extension);
+            }
+
+            if (UserThemesDirectory == null)
+            {
+                throw new NotSupportedException("There is no directory for custom user themes in portable mode");
```

Comment:
```suggestion
                throw new InvalidOperationException("There is no directory for custom user themes in portable mode");
```
https://docs.microsoft.com/dotnet/api/system.notsupportedexception#related-exception-types